### ISSUE TITLE
Can it upload video?

### ISSUE THREAD

**Issue by VietDucPhan** [andrewgouin/react-native-cloudinary#2]
Hi,

Thanks for the great work.
I have a question. Can I use this to upload video?

**Comment by andrewgouin** [comment]
@VietDucPhan Yes, it is capable of uploading any kind of media, and it can upload multiple files concurrently.
